  File "C:\Users\gayat\OneDrive\Desktop\code\projects\dbms_project\server\routes.py", line 56, in signin
    name = i.name
AttributeError: 'user_credentials' object has no attribute 'name'